assets/javascripts/discourse/initializers/extend-for-translate-button.js

```diff
@@ -23,7 +24,33 @@ function initializeTranslation(api) {
     );
   }
 
-  if (currentUser) {
+  if (
+    siteSettings.experimental_topic_translation &&
+    (currentUser || siteSettings.experimental_anon_language_switcher)
+  ) {
+    api.renderInOutlet("topic-navigation", ShowOriginalContent);
+    api.decorateCookedElement((cookedElement, helper) => {
+      if (helper) {
+        const translatedCooked = helper.getModel().get("translated_cooked");
+        if (translatedCooked) {
+          cookedElement.innerHTML = translatedCooked;
+        } else {
```

Comment:
I'm more familiar with ruby so I genuinely am not sure, but in this case do we even want the `else` for now?